### PR TITLE
D8CORE-4236: fixing the event date alignment on cards and lists of cards.

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- correcting the alignment of the event date when in a card or in a paragraph list. 
- card on the event node will also be correct to a single line instead of stacked.
- only the event list should have stacked dates when the event spans days.

# Review By (Date)
- 7/2

# Criticality
- Effects Events on all the sites when it has a date that spans over time

# Urgency
- normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Checkout this branch: https://github.com/SU-SWS/stanford_events/pull/76
2. Rebuild Cache and import config `drush cr ; drush ci`
3. add several events that spans days
4. Verify that the /events page dates will stack.
5. Add list paragraphs of card and default types and verify the date is in a line  and not stacked. 

Screen Shot 2021-06-29 at 8 32 48 AM
Screen Shot 2021-06-29 at 8 32 08 AM

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [x] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- All themes using events.
- I tested locally on: SBT, Undergrad, and SAA

# Associated Issues and/or People
- [D8CORE-4326](https://stanfordits.atlassian.net/browse/D8CORE-4236)
- https://github.com/SU-SWS/stanford_profile_helper/pull/124
- https://github.com/SU-SWS/stanford_events/pull/76

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss [hunk 1]
```diff
@@ -320,6 +320,11 @@
       }
     }
   }
+
+  .su-event-list-item__date {
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss:324]: Why is this necessary if the same styles are applied in [stanford_events](https://github.com/SU-SWS/stanford_events/pull/76/files#diff-0cde1a1f3737bd25dc63d5c13d361da7c2a74abf976f2ee9bb957fd19f4a279bR4)

I followed the steps and with only the stanford_events PR I believe it works for me.

Comment (jenbreese) [reply modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss:324]: it is necessary for the default list version in a paragraph. If you add a list in the default style, you would see this.
image

I thought about adding the `.ptype-stanford-lists` to the events file, but we've talked before about keeping them separate.

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss:324]: What about just making it less specific. if i remove `.su-card` from the [scss](https://github.com/SU-SWS/stanford_events/pull/76/files#diff-0cde1a1f3737bd25dc63d5c13d361da7c2a74abf976f2ee9bb957fd19f4a279bR3) it applies the same styles to both displays.

Comment (jenbreese) [reply modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss:324]: True but then it does this on the /events list pages. 
image
Better to override it just on the list page? I could look at it that way too.

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/component/node/stanford_page.scss:324]: Oh gotcha. This is really why we need the `@container` css if it were widely available. Darn you CSS!

**Review by pookmish — APPROVED** [review]
